web/client/plugins/FeatureEditor.jsx

```diff
@@ -144,6 +144,7 @@ const FeatureDock = (props = {
             {getDialogs(props.tools)}
             <Grid
                 onMount={props.onMount}
+                showFilteredObject={props.showFilteredObject}
                 editingAllowedRoles={props.editingAllowedRoles}
```

Comment:
we could remove this props because they are not used anymore for grid
```
onMount={props.onMount}
showFilteredObject={props.showFilteredObject}
```
they have been moved to the lifecycle of plugin